tests-behat/vue.feature

```diff
@@ -9,6 +9,5 @@ Feature: Vue
     Then I hide js modal
 
     When I fill in "atk-vue-search" with "united kingdom"
-# needed because debouce is used
-    Then I sleep 250 ms
+    Then I sleep 350 ms
```

Comment:
there should be no need for waiting in testing, see https://github.com/atk4/ui/pull/1430/files#r478697880

Reply:
I add an option for settings debounce time in component and set proper test for it.